packages/ui/src/banner-manager.tsx

```diff
@@ -74,18 +74,16 @@ export class BannerManager {
   constructor(private options: BannerManagerOptions) {
     this._options = options;
 
-    const playerConfig =
-      options.corePlayer &&
-      options.corePlayer.config &&
-      options.corePlayer.config.contrib &&
-      options.corePlayer.config.contrib.ui &&
-      options.corePlayer.config.contrib.ui.kitchenSink
-        ? options.corePlayer.config.contrib.ui.kitchenSink
-        : {};
+    const managerConfig = ObjectUtils.get(
+      this._options.corePlayer,
+      'config.contrib.ui.banner',
+      DefaultBannerConfig
```

Comment:
the default here is empty object, the actual default is assigned later using `mergeDefaults`